shared/java/javacommon.gradle

```diff
@@ -155,6 +155,13 @@ protobuf {
     }
     generateProtoTasks {
         all().configureEach { task ->
+            if (project.hasProperty('skipproto')) {
+                task.builtins {
+                    cpp {}
+                    remove java
+                }
+                return
+            }
```

Comment:
How is this used?

Reply:
This was needed because wpiutil only had protobufs in the test, and the google protobufs were erroring for some reason. 

I’ll be removing all of this Infra in a future pr. This pr was just already so big it wasn’t worth it.